packages/stream_chat_flutter/lib/src/utils/extensions.dart

```diff
@@ -353,6 +366,25 @@ extension MessageX on Message {
     return copyWith(text: messageTextToRender);
   }
 
+  /// Returns an approximation of message size
+  double roughMessageSize(double? fontSize) {
+    var messageTextLength = min(text!.biggestLine().length, 65);
+
+    if (quotedMessage != null) {
+      var quotedMessageLength =
+          (min(quotedMessage!.text?.biggestLine().length ?? 0, 65)) + 6;
```

Comment:
We break the line after, more or less, 65 characters. So it can't be bigger then this.